### DISCUSSION TITLE
Downloaded depots cannot be opened? MacOS Catalina

### DISCUSSION THREAD

**Discussion by polyprop** [SteamRE/DepotDownloader#194] [Q&A]
MacOS Catalina 10.15.7 

Have successfully downloaded the depots of 2 different Steam games. However, when I try to launch either of them, I get a pop-up saying "The application "<name>" can't be opened". Is there an additional step I'm missing that's needed to let the downloaded games run?

**Comment by ghost** [comment]
You need to change the permission of the executable:
```
chmod +x <Bundle>.app/Contents/MacOS/<Game>
```
Replace `<Bundle>` with your app bundle name and `<Game>` with the executable name.
